tests/headless/test_cdn.py

```diff
@@ -251,8 +256,10 @@ def test_cached(base_url, is_behind_cdn, slug):
     'slug', ['/files/2767/hut.jpg',
              '/@api/deki/files/3613/=hut.jpg',
              '/en-US/dashboards/localization'])
-def test_cached_301(base_url, is_behind_cdn, slug):
+def test_cached_301(base_url, is_behind_cdn, is_local_url, slug):
     """Ensure that these requests that should return 301 are cached."""
+    if is_local_url and any(slug.startswith(p) for p in ('/files/', '/@api/')):
+        pytest.xfail('not yet available in the sample database')
```

Comment:
Files won't be part of the sample database either. Stage has these files because we restored an anonymized backup and synced attachments at one point.